src/components/MemberDetails.tsx

```diff
@@ -125,11 +134,7 @@ export function MemberDetails({ member }: MemberDetailsProps) {
             <Serif size="4" weight="semibold">
               Manager:
             </Serif>
-            <RouterLink
-              href={"/member/[member]"}
-              as={`/member/${normalizeParam(manager.name)}`}
```

Comment:
Next.js 10 got rid of the need to have these `as` props when you're linking to a dynamic page. Thank goodness.